config/settings.properties

```diff
@@ -6,3 +6,6 @@ username=user
 
 # Password for our account
 password=pass
+
+# Destination directory for downloaded WARC files
+outputDir=/var/downloadedWarcFiles
```

Comment:
I presume there is also a PR in to do this in shared_configs project, or at least an issue for it?